realsense/scene_perception/js/scene_perception_api.js

```diff
@@ -37,6 +37,34 @@ var ScenePerception = function(objectId) {
     return {width: width, height: height, data: preview};
   };
 
+  function wrapGetVolumePreviewReturn(data) {
+    const BYTES_PER_INT = 4;
+    const BYTES_PER_FLOAT = 4;
+    const BYTES_OF_RGBA = 4;
```

Comment:
I think it would be good to consolidate all these constants into module level instead of re-defined in each methods. Could you please help do that? @jondong Thanks!